herd/libdir/aarch64memattrs.cat

```diff
@@ -41,100 +41,61 @@
  * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  *)
 
+let TTD = PTE
+
+procedure conflict(attr, set) =
+  let diff = attr & (set \ attr)
+  assert empty diff as Invalid-Attribute
+end
```

Comment:
Unless I am missing something this check will always succeed, whatever the sets `attr` and `set` are.

Reply:
Ofc you are right Luc. I think there is a simpler test which solves the problem:
`iWB & iWT & iNC` should be empty